logtail/logtail.go

```diff
@@ -468,10 +506,34 @@ func (l *Logger) encodeText(buf []byte, skipClientTime bool, level int) []byte {
 	b := make([]byte, 0, len(buf)+overhead)
 	b = append(b, '{')
 
-	if !skipClientTime {
-		b = append(b, `"logtail": {"client_time": "`...)
-		b = now.AppendFormat(b, time.RFC3339Nano)
-		b = append(b, "\"}, "...)
+	if includeLogtail {
+		b = append(b, `"logtail": {`...)
+		needsComma := false
+		if !skipClientTime {
+			b = append(b, `"client_time": "`...)
+			b = now.AppendFormat(b, time.RFC3339Nano)
+			b = append(b, '"')
+			needsComma = true
+		}
+		if instanceKey != nil {
+			if needsComma {
+				b = append(b, ',')
+			}
+			b = append(b, `"instance_key": "`...)
```

Comment:
proc_key, proc_seq ?

"instance" already has a meaning and I'd prefer to not overload that.

Reply:
Ended up with `proc_id` and `proc_seq`